src/main/java/com/microsoft/sqlserver/jdbc/ConfigurableRetryLogic.java

```diff
@@ -357,10 +357,19 @@ private static LinkedList<String> readFromFile(String connectionStringProperty)
             if (CONFIGURABLE_RETRY_LOGGER.isLoggable(java.util.logging.Level.FINER)) {
                 CONFIGURABLE_RETRY_LOGGER.finest("File not found at path - \"" + filePath + "\"");

```

Comment:
Not your code, but the logging level is inconsistent between the IF and log action. I'd also go FINE here, not FINER or FINEST.
```suggestion
            if (CONFIGURABLE_RETRY_LOGGER.isLoggable(java.util.logging.Level.FINE)) {
                CONFIGURABLE_RETRY_LOGGER.fine("File not found at path - \"" + filePath + "\"");
```